### ISSUE TITLE
🐛BPMN-Studio breaks when altering CORS options in Runtime 

### ISSUE THREAD

**Issue by Vyperus** [process-engine/bpmn-studio#1457]
#### Describe your issue

I am working on project where I need to use the event system of the runtime via the consumer client.
To avoid CORS problems i changed the options like this:

<img width="404" alt="Bildschirmfoto 2019-04-01 um 10 59 40" src="https://user-images.githubusercontent.com/17248802/55315771-49edcb00-546d-11e9-8952-259ceac2ee98.png">

This way my new application is working as expected.
But if I now connect BPMN-Studio with this PE the following errors are occurring:

1. The Dashboard is broken:
<img width="1412" alt="Bildschirmfoto 2019-04-01 um 11 02 27" src="https://user-images.githubusercontent.com/17248802/55315925-a7821780-546d-11e9-962d-cbd5f11f5442.png">

2. The Inspect Correlation is bot showing running correlations:
<img width="1412" alt="Bildschirmfoto 2019-04-01 um 11 02 27" src="https://user-images.githubusercontent.com/17248802/55316011-caacc700-546d-11e9-9bc9-84f065a5cf4e.png">

3. The LET doesn't function anymore:
<img width="1412" alt="Bildschirmfoto 2019-04-01 um 11 03 51" src="https://user-images.githubusercontent.com/17248802/55316057-e2844b00-546d-11e9-970f-d31235baa16c.png">

#### How can others reproduce the issue?

See above.

#### What's your setup

- BPMN-Studio Version: 5.1.2

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that I think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

**Comment by alexanderkasten** [comment]
Maybe you must reconfigure the Requests here:

https://github.com/process-engine/bpmn-studio/blob/develop/src/modules/fetch-http-client/HttpFetchClient.ts

**Comment by Vyperus** [comment]
I debugged this right now and here is what I found:

This behavior only happens if you use the PE Runtime on the develop branch.
Doesn't happen on master.

On develop it totally doesn't matter what settings you use it is always broken.

Here ist my step by step guide to reproduce this:
1) Start the PE on develop.
2) Connect BPMN-Studio.
3) Deploy and try to start a process.
4) Go to Dashboard.

This is log when its broken:
<img width="419" alt="Bildschirmfoto 2019-04-02 um 08 40 45" src="https://user-images.githubusercontent.com/17248802/55381816-52eda380-5524-11e9-87e1-a8380a38f95d.png">
Notice that "processModels" is undefined.

This is log when i do the same but with the master branch:
<img width="431" alt="Bildschirmfoto 2019-04-02 um 08 42 49" src="https://user-images.githubusercontent.com/17248802/55381842-639e1980-5524-11e9-8629-f1c1e46fd354.png">
See that "processModels" is not undefined here.

Due to this behavior this is most likely not a bug in BPMN-Studio so feel free to move this Issue ;)
